en/admin/config.rst

```diff
@@ -2303,6 +2311,20 @@ The following are other parameters. The type and value range for each parameter
 
 **vacuum_ovfp_check_duration** specifies the duration for which data related to the count of index overflow pages, gathered by vacuum threads, is retained. Data that remains unchanged within the specified duration will be automatically removed. The unit of it's value is minutes.
 
+**deduplicate_key_level**
+
+ Specifies the *deduplicate level* value implicitly specified in the index creation statement. Default is -1. See :doc:`/sql/schema/table_stmt`\ and :doc:`/sql/schema/index_stmt`.
```

Comment:
The verb `specify` is repeated redundantly. And It seems to be better to add a reference to DEDUPLICATE.
 
```suggestion
 **deduplicate_key_level** determines the automatic inclusion and value setting of the WITH DEDUPLICATE statement within the index creation statement. For details on DEDUPLICATE, see :ref:`deduplicate_overview`. The default is -1(which means that the WITH DEDUPLICATE is not included implicitly).
```